### PR TITLE
Docs: use `username` instead of `user` in Credentials

### PR DESCRIPTION
**Pull request by oleurud**
Fix Credentials parameter name in docs: `username` instead of `user`

### CODE REVIEW

File: docs/developer-center/_internal/reference/07-CartoContext.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ There are two ways of authenticating against a CARTO account:
 
     ```python
     from cartoframes import Credentials
-    creds = Credentials(user='eschbacher', key='abcdefg')
```
Comment (andy-esch) [inline docs/developer-center/_internal/reference/07-CartoContext.md]: This has been replaced (for now) by the readthedocs version which has the correct `username` kwarg:
https://cartoframes.readthedocs.io/en/stable/cartocontext.html#cartocontext

File: docs/developer-center/guides/01-Quickstart.md [hunk 2]
```diff
@@ -71,15 +71,15 @@ cc = CartoContext(
 
 ```python
 from cartoframes import Credentials, CartoContext
-creds = Credentials(user='<your_user_name>', key='<your_api_key>')
+creds = Credentials(username='<your_user_name>', key='<your_api_key>')
```
Comment (andy-esch) [inline docs/developer-center/guides/01-Quickstart.md]: Thanks, great catch! Once we merge this into master, we'll need to merge master into [this PR](https://github.com/CartoDB/cartoframes/pull/558/files#diff-e023f6411ac78a05e315d320aca2d2b1).

Comment (oleurud) [reply docs/developer-center/guides/01-Quickstart.md]: I was trying some examples and I found this problem. I did know you had created another one, let me move the commit to your initial PR and I will close this one

**Comment by oleurud** [comment]
Commit moved to bigger PR https://github.com/CartoDB/cartoframes/pull/558 with similar things

**Comment by andy-esch** [comment]
perfect, thanks again @oleurud

**Pull request closed without merging**